ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -291,6 +301,13 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     });
   });
 
+  const solanaAccountCreated = useMemo(() => {
+    return selectedCaipAccountAddresses.some((caipAccountId) => {
```

Comment:
this logic doesn't actually identify if a Solana account is created, only whether it is selected.

Reply:
Admittedly this is a very unlikely user path but we should probably just look at the correct state to identify whether a solana account is created or not:

https://github.com/user-attachments/assets/14b62583-7d0e-44d8-99d9-c84ae487f4fd

Reply:
Good catch! I just pushed the fix